games/godot/Portfile

```diff
@@ -37,18 +37,28 @@ if {$subport eq ${name}} {
 }
 
 subport ${name}-3.2 {
-    github.setup    godotengine ${name} 3.2.3 "" -stable
+    PortGroup       obsolete 1.0
 
-    checksums       rmd160  4ff4f9c4798d67cf6cf4d63c38bed84d400daa8e \
-                    sha256  6cd925cc4c23e2c4b0bf36458a83e03484184d8d5311b8eb2339319df7c0c26c \
-                    size    22460848
+    version         3.2.3
+    replaced_by     ${name}
 
-    long_description-append \
-                    \n\n(Note: Godot ${version} is the last version of \
-                    Godot that is officially supported on macOS \
-                    10.9-10.11.)
+    # 2021-12-09: I believe that it would be safe to completely remove
+    # this subport without needing to wait for the traditional one-year
+    # waiting period. The only reason I originally created this subport
+    # was because Godot 3.3+ was failing to compile for macOS <= 10.10.
+    # Now that we have been able to get the source code to compile and
+    # run successfully on older macOSes, this subport is no longer
+    # necessary. In addition, based on the statistics, it looks like
+    # no one has ever installed this subport.
```

Comment:
just replace this with the date the subport can be removed please

Reply:
See my comment above.

Reply:
to me all these redundant comments make a Portfile harder to read... but sure, if you really want it - you're the maintainer